tests/utils/test_time_utils.py

```diff
@@ -58,97 +60,47 @@ def test_get_year_remaining(self):
                 time_remaining = time.get_years_remaining(
                     test_case["market_time"], test_case["mint_time"], test_case["num_position_days"]
                 )
-                np.testing.assert_almost_equal(
-                    time_remaining, test_case["expected_result"], err_msg=f"unexpected time remaining {time_remaining}"
+                self.assertAlmostEqual(
+                    time_remaining,
+                    test_case["expected_result"],
+                    delta=self.APPROX_EQ,
+                    msg=f"unexpected time remaining {time_remaining}",
                 )
 
-    def test_norm_days(self):
```

Comment:
are these deprecated because we use block_time now?

Reply:
We didn't really use some of them. Like only once. And the normalizing one was causing confusion when it was used so I modified the code to just do the division when needed